### PR TITLE
商品一覧の検索フォームをレスポンシブデザインに変更しました

### PR DESCRIPTION
**Pull request by SekiMoemi**

### CODE REVIEW

File: src/main/java/jp/co/example/ecommerce_a/common/GlobalExceptionHandler.java [hunk 1]
```diff
@@ -0,0 +1,21 @@
+package jp.co.example.ecommerce_a.common;
```
Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/common/GlobalExceptionHandler.java:1]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.naming.PackageNameCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Package name 'jp.co.example.ecommerce_a.common' must match pattern '^[a-z]+(\.[a-z][a-z0-9]*)*$'.

File: src/main/java/jp/co/example/ecommerce_a/common/GlobalExceptionHandler.java [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+
+@ControllerAdvice
```
Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/common/GlobalExceptionHandler.java:8]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.javadoc.MissingJavadocTypeCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Missing a Javadoc comment.

File: src/main/java/jp/co/example/ecommerce_a/common/GlobalExceptionHandler.java [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+public class GlobalExceptionHandler {
+
+	private static final Logger logger = LoggerFactory.getLogger(GlobalExceptionHandler.class);
```
Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/common/GlobalExceptionHandler.java:11]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/common/GlobalExceptionHandler.java:11]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'member def modifier' has incorrect indentation level 8, expected level should be 2.

File: src/main/java/jp/co/example/ecommerce_a/common/GlobalExceptionHandler.java [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+
+	@ExceptionHandler(Throwable.class)
```
Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/common/GlobalExceptionHandler.java:13]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/common/GlobalExceptionHandler.java:13]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'method def modifier' has incorrect indentation level 8, expected level should be 2.

Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/common/GlobalExceptionHandler.java:13]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.javadoc.MissingJavadocMethodCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Missing a Javadoc comment.

File: src/main/java/jp/co/example/ecommerce_a/common/GlobalExceptionHandler.java [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+	public String handleConnectionError(Exception e) {
```
Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/common/GlobalExceptionHandler.java:14]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

File: src/main/java/jp/co/example/ecommerce_a/common/GlobalExceptionHandler.java [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+
+		logger.error(e.getMessage(), e);
```
Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/common/GlobalExceptionHandler.java:16]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/common/GlobalExceptionHandler.java:16]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'method def' child has incorrect indentation level 16, expected level should be 4.

File: src/main/java/jp/co/example/ecommerce_a/common/GlobalExceptionHandler.java [hunk 7]
```diff
@@ -0,0 +1,21 @@
...
+
+		return "error/500";
```
Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/common/GlobalExceptionHandler.java:18]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/common/GlobalExceptionHandler.java:18]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'method def' child has incorrect indentation level 16, expected level should be 4.

File: src/main/java/jp/co/example/ecommerce_a/common/GlobalExceptionHandler.java [hunk 8]
```diff
@@ -0,0 +1,21 @@
...
+	}
```
Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/common/GlobalExceptionHandler.java:19]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/common/GlobalExceptionHandler.java:19]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'method def rcurly' has incorrect indentation level 8, expected level should be 2.

**Review by yamada-kensyu — APPROVED** [review]
承認します

**Pull request merged**